### DISCUSSION TITLE
I want to use BLE with Android 13 (API 33) Xamarin. What version of Shiny should I use?

### DISCUSSION THREAD

**Discussion by rvalitov** [shinyorg/shiny#1309] [Q&A]
It seems that the stable version of Shine BLE supports Android 12 only. Is there a way to use it with Android 13?

**Comment by aritchie** [comment]
It doesn't "only" support Android 12.  It supports Android 12 and up.  The target is 12.  The latest stable works fine for classic xamarin.  The v3 beta support net7 targets and Android 13+
